tests/integration/test_services_auto_tagging_copilot_feedback.py

```
@@ -0,0 +1,1217 @@
+        assert "preferred_locations" in history
+
+    def test_clear_old_feedback(self, tmp_path: Path) -> None:
+        from file_organizer.models.suggestion_types import SuggestionType
+        from file_organizer.services.suggestion_feedback import FeedbackEntry, SuggestionFeedback
+
+        fb = SuggestionFeedback(feedback_file=tmp_path / "feedback.json")
+        old_entry = FeedbackEntry(
+            suggestion_id="old1",
+            suggestion_type=SuggestionType.MOVE,
+            action="accepted",
+            file_path="/tmp/old.txt",
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/old.txt"

(S108)